aten/src/ATen/native/cpu/DepthwiseConvKernel.cpp

```diff
@@ -22,6 +22,7 @@ struct Arguments final {
   // Output layer dimensions
   int64_t out_rows;
   int64_t out_cols;
+  int64_t out_channels;
```

Comment:
Why add it here as you don't actually pass it to the impl and only use it on the top level function?

Reply:
No particular reason, just keeping the pattern, can make it a local var if you prefer.

Reply:
It just creates a larger struct and pass it around for no reason.
But I agree it makes the code easier to read and it should be pretty insignificant impact so I'm happy to keep it as-is.